openshift/admin-console.app.yaml

```diff
@@ -74,6 +74,11 @@ objects:
               configMapKeyRef:
                 name: admin-console
                 key: auth.url
+          - name: ADMIN_TENANT_URL
+            valueFrom:
+              configMapKeyRef:
+                name: admin-console
+                key: tenant.url
```

Comment:
Also add it to https://github.com/fabric8-services/admin-console/blob/master/openshift/admin-console.config.yaml

Reply:
done in d277562